src/averageAges.js

```diff
@@ -16,10 +16,20 @@
  */
 function calculateMenAverageAge(people, century) {
   // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => (
+    person.sex === 'm'
+    && (century
+      ? Math.ceil(person.died / 100) === century
+      : true
+    )
+  ));
+
+  const personAges = men.map(man => man.died - man.born);
+  const totalAges = personAges.reduce((prevValue, currentValue) => (
+    prevValue + currentValue
+  ));
```

Comment:
You can also calculate age value in reduce method for avoiding redundant map method